tests/controls/gridtest.cpp

```diff
@@ -1187,6 +1181,9 @@ TEST_CASE_METHOD(GridTestCase, "Grid::ResizeScrolledHeader", "[grid]")
 
     wxYield();
     sim.MouseMove(point);
+#ifdef __WXGTK20__
+    sim.MouseMove(point);
+#endif
```

Comment:
This is really weird, but I guess you don't know why is this needed neither?

Reply:
I did some tracing and found that, for a given window, gtk_window_motion_notify_callback is not called at all due to a fake motion unless the pointer is already on the same window. and unless i am missing something, without motion event, wxGrid won't be able to begin a drag operation. this is the reason for the second call to MouseMove(). sadly, even if it works now locally, i failed to make it work on Travis CI !

Reply:
Thanks for the explanation!

Ideal would be to take care of this inside `MouseMove()` implementation itself, i.e. check if the mouse is not in the window there and generate 2 events in this case. And if we can't do it there, we should at least do it in `MouseDragDrop()` where it should definitely be possible and where it would always be needed.

Reply:
Done!